src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NewNotifichePagoPATest.java

```diff
@@ -35,17 +35,16 @@ public void creoUnaNotificaConUnDestinatarioEUnDocumento() throws RestNotificati
             ArrayList<Document> documents = new ArrayList<>();
             documents.add(new Document());
             NewNotification notification = new NewNotification(DataPopulation.generatePaProtocolNumber(), "Pagamento Rata IMU", recipients, documents, PhysicalCommunicationTypeEnum.AR_REGISTERED_LETTER, "123456A", NotificationFeePolicyEnum.FLAT_RATE);
-
             NewNotificationResponse response = restNotification.newNotificationWithOneRecipientAndDocument(notification);
 
             if (response != null) {
```

Comment:
do condition with responseCode

Reply:
Please check the method `newNotificationWithOneRecipientAndDocument()` you'll notice what you asked